server/Html.js

```diff
@@ -71,6 +71,7 @@ class Html extends Component {
         <body>
           <div id="root" />
           <script dangerouslySetInnerHTML={{ __html: initialStateHtml }} />
+          <script src="https://cdn.polyfill.io/v2/polyfill.min.js?features=Intl.~locale.fr,Intl.~locale.pt" />
```

Comment:
- [ ] Are you intentionally loading this for French and Portuguese? Finnish, Swedish and English would make more sense.